lib/capistrano/recipes/deploy/strategy/jenkins_artifact.rb

```diff
@@ -70,9 +81,8 @@ def deploy!
       }.to_s
     end
 
-    build_num = client.job.get_last_successful_build_number(dir_name)
-    timestamp = client.job.get_build_details(dir_name, build_num)['timestamp']
-    deploy_at = Time.at(timestamp / 1000)
+    last_successful_build = client.job.get_last_successful_build(dir_name)
```

Comment:
If another build succeeds after `:artifact_url` determined but before this line, those two may point different build?